ods_ci/tests/Resources/Page/DistributedWorkloads/DistributedWorkloads.resource

```diff
@@ -128,6 +128,24 @@ Prepare Training Operator E2E Test Suite
     Enable Component    trainingoperator
     Wait Component Ready    trainingoperator
 
+Prepare Training Operator SDK Test Suite
```

Comment:
Can utilize existing keyword "Prepare Training Operator E2E Test Suite    ${KFTO_BINARY_NAME}" and avoid repetition of code

Reply:
Yes, true. I will reuse it.